onnxscript/ir/_core.py

```diff
@@ -1553,7 +1673,7 @@ def __str__(self) -> str:
         inputs_text = ",\n".join(str(x) for x in self.inputs)
         outputs_text = ",\n".join(str(x) for x in self.outputs)
         attributes_text = ",\n".join(
-            attr.name + f": {attr.type}" + f"= {attr.value}" * (attr.value is None)
+            attr.name + f": {attr.type}" + f" = {attr.value}" * (attr.value is None)
```

Comment:
Why not using a single string instead of a concatenation?

Reply:
Good point. Let me try that

Reply:
I merged the first parts. Seems a little hard to merge `+ f" = {attr.value}" * (attr.value is None)`